integration/e2ecortex/services.go

```diff
@@ -350,6 +350,26 @@ func NewAlertmanager(name string, flags map[string]string, image string) *Cortex
 	)
 }
 
+func NewAlertmanagerWithTLS(name string, flags map[string]string, image string) *CortexService {
```

Comment:
To my understanding the only difference between this and `NewAlertmanager` is the readiness probe. What's the problem making the readiness probe working with HTTPS too?

Reply:
The only reason I didn't do this was because it required a change to the function parameters in the `e2e` package that were rather far reaching. Would it be acceptable to do this in a separate PR?

Reply:
Sure. I haven't checked the complexity and it's not a big deal.